js/ui/drop_down_button.js

```diff
@@ -519,14 +522,20 @@ const DropDownButton = Widget.inherit({
                 break;
             case 'displayExpr':
                 this._compileDisplayGetter();
+                this._setListOption(name, value);
+                this._updateActionButton(this.option('selectedItem'));
                 break;
             case 'keyExpr':
                 this._compileKeyGetter();
+                this._setListOption(name, value);
                 break;
             case 'buttonGroupOptions':
                 this._innerWidgetOptionChanged(this._buttonGroup, args);
                 break;
             case 'dropDownOptions':
+                if(args.value.visible !== undefined) {
+                    delete args.value.visible;
```

Comment:
`instance.option('dropDownOptions.visible', true)` raises `valueChanged` event with args: 
```
{
  name: "dropDownOptions",
  fullName: "dropDownOptions.visible",
  value: true
}

Reply:
fixed it, thanks